applications/accounts-api/backend/accounts_api/controllers/users_controller.py

```diff
@@ -38,6 +38,22 @@ def get_user(userid):  # noqa: E501
         return "User not found", 404
 
 
+def get_users(query={}):
+    """get all users
+
+    :param query: user filter
+    :type query: str
+
+    :rtype: {}
+    """
+
+    try:
+        return {'users': user_service.get_users(query)}
+    # TODO: better handle exceptions
+    except Exception as e:
+        return "Error"
```

Comment:
Why not just leave the exception go instead?

Reply:
if you leave the exception go, the default is a 500 error

Reply:
Done---is that what you meant?